lib/main_common.pm

```diff
@@ -234,6 +235,14 @@ sub is_kernel_test {
         || get_var('TRINITY'));
 }
 
+sub get_ltp_tag {
+    my $tag = get_var('LTP_RUNTEST_TAG');
+    if (!defined $tag) {
+        $tag = get_var('DISTRI') . '-' . get_var('VERSION') . '-' . get_var('ARCH') . '-' . get_var('BUILD') . '-' . get_var('FLAVOR') . '-' . get_var('LTP_RELEASE', 'master') . '@' . get_var('MACHINE');
```

Comment:
although DISTRI, VERSION, ARCH are variables that are always there, I would still use get_required_var(), maybe not the case for LTP_RELEASE

Reply:
No, I don't want to whole LTP testing fail just because these variables are not present. We tried to avoid failing `install_ltp` as much as we can. This tag is a prefix for filename, in a worst case there will be files overwritten (which was a problem before).